staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy.go

```diff
@@ -56,11 +64,17 @@ type proxyHandler struct {
 }
 
 func (r *proxyHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
-	proxyRoundTripper, err := r.getRoundTripper()
-	if err != nil {
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	handlingInfo := r.handlingInfo.Load().(proxyHandlingInfo)
```

Comment:
> this will panic if nothing has been stored in the value yet, handle not-ok casts with not found error or something

Private struct.  It's always initialized before it's exposed.